### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by tjsavage** [webcomponents/beta#317]
We should get a thorough a11y review before we open this up at the Summit. When do you think we might be a good state to run through with an audit?

**Comment by samuelli** [comment]
@tjsavage: if you can set this up for early next week that'll be stellar.

**Comment by tjsavage** [comment]
Will ask Laura if she'll have some time next week.

**Comment by Nevraeka** [comment]
We fail lighthouse tests for color contrast. 
✘ Elements must have sufficient color contrast
```css
#container > section:nth-of-type(1) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(1) > .container.style-scope.collection-card > a > div
#container > section:nth-of-type(1) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(1) > .container.style-scope.collection-card > div > div > .style-scope.collection-card
#container > section:nth-of-type(1) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(2) > .container.style-scope.collection-card > a > div
#container > section:nth-of-type(1) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(2) > .container.style-scope.collection-card > div > div > .style-scope.collection-card
#container > section:nth-of-type(1) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(3) > .container.style-scope.collection-card > a > div
#container > section:nth-of-type(1) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(3) > .container.style-scope.collection-card > div > div > .style-scope.collection-card
#container > section:nth-of-type(2) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(1) > .container.style-scope.collection-card > a > div
#container > section:nth-of-type(2) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(1) > .container.style-scope.collection-card > div > div > .style-scope.collection-card
#container > section:nth-of-type(2) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(2) > .container.style-scope.collection-card > a > div
#container > section:nth-of-type(2) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(2) > .container.style-scope.collection-card > div > div > .style-scope.collection-card
#container > section:nth-of-type(2) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(3) > .container.style-scope.collection-card > a > div
#container > section:nth-of-type(2) > .featured-row.loader.style-scope.catalog-home > collection-card:nth-of-type(3) > .container.style-scope.collection-card > div > div > .style-scope.collection-card
#getting-started > a:nth-of-type(1) > div
#gitter > div > p
#gitter > a > .style-scope.catalog-home
```

**Comment by Nevraeka** [comment]
If we have a standard palette, I can do a PR for these. What is the styleguide (if any) we are using?

**Comment by samuelli** [comment]
Oops, sorry I never replied @Nevraeka!

Desktop screens: https://invis.io/5C8AYM7KM
Mobile screens: https://invis.io/PX8LYWL96
Sketch file: https://www.dropbox.com/s/zl9ty338im3n0h9/customelements-hf_r2a.sketch?dl=0

I've tuned almost all contrast failures using AXE. There's a couple of outstanding ones which are harder to fix, like the "publish element" button, and "demo" on slight grey background text. All the links have been fixed to meet AA.

**Comment by samuelli** [comment]
I think these should all be fixed now.
